content/en/tracing/trace_collection/library_config/nodejs.md

```diff
@@ -48,6 +48,15 @@ The version number of the application.
 **Default**: `{}`<br>
 Set global tags that are applied to all spans and runtime metrics. When passed as an environment variable, the format is `key:value,key:value`. When setting this programmatically, the format is `tracer.init({ tags: { foo: 'bar' } })`.
 
+`DD_TRACE_HEADER_TAGS`
+: **Configuration**: `headerTags` <br>
+Accepts a map of case-insensitive header keys to tag names and automatically applies matching header values as tags on traces. Also accepts entries without a specified tag name that are automatically mapped to tags of the form `http.request.headers.<header-name>` and `http.response.headers.<header-name>`.<br><br>
```

Comment:
As mentioned out of band try to word it so that the tag name is optional and add a full example, like `User-ID:userId,Request-ID`.